modules/otmBidAdapter.js

```diff
@@ -132,15 +136,42 @@ function _getBidFloor(bid) {
     return bid.params.bidfloor ? bid.params.bidfloor : 0;
   }
 
-  let floor = bid.getFloor({
-    currency: DEF_CUR,
+  const floor = bid.getFloor({
+    currency: DEFAULT_CURRENCY,
     mediaType: '*',
     size: '*'
   });
-  if (isPlainObject(floor) && !isNaN(floor.floor) && floor.currency === DEF_CUR) {
+  if (isPlainObject(floor) && !isNaN(floor.floor) && floor.currency === DEFAULT_CURRENCY) {
     return floor.floor;
   }
   return 0;
 }
 
+/**
+ * Get top level origin
+ * @return {string}
+ */
+function getTopDomain() {
+  try {
+    return window.top.location.host
```

Comment:
you can't do any of this; you have to use our methods.

Reply:
changed to `getWindowTop()`